Description: Fix ldconfig-symlink-referencing-wrong-file
 Let libcktee.so.0 link to libcktee.so.0.x.x
Author: Ying-Chun Liu (PaulLiu) <paulliu@debian.org>
Last-Update: 2022-02-05

Index: optee-client-3.16.0/libckteec/Makefile
===================================================================
--- optee-client-3.16.0.orig/libckteec/Makefile
+++ optee-client-3.16.0/libckteec/Makefile
@@ -61,7 +61,7 @@ $(OUT_DIR)/$(LIBCKTEEC_AR_LIBRARY): $(LI
 
 libckteec:
 	$(VPREFIX)ln -sf $(LIB_MAJ_MIN_PAT) $(OUT_DIR)/$(LIB_MAJ_MIN)
-	$(VPREFIX)ln -sf $(LIB_MAJ_MIN) $(OUT_DIR)/$(LIB_MAJOR)
+	$(VPREFIX)ln -sf $(LIB_MAJ_MIN_PAT) $(OUT_DIR)/$(LIB_MAJOR)
 	$(VPREFIX)ln -sf $(LIB_MAJOR) $(OUT_DIR)/$(LIB_NAME).so
 
 ################################################################################
